config/opal_check_package.m4

```diff
@@ -134,6 +134,10 @@ AC_DEFUN([_OPAL_CHECK_PACKAGE_LIB], [
                   test "$ac_cv_search_$3" != "none required"],
                  [$1_LIBS="$ac_cv_search_$3 $4"],
                  [$1_LIBS="$4"])
+            # libnl and libnl3 are known *not* to coexist
```

Comment:
Minor: update the comment to "libnl v1 and v3 are known to cause a crash if they are both linked into the same process."